dm/config/subtask.go

```diff
@@ -44,13 +44,41 @@ var (
 	defaultMaxAllowedPacket = 64 * 1024 * 1024 // 64MiB, equal to TiDB's default
 )
 
+// RawDBConfig contains some low level database config
+type RawDBConfig struct {
+	MaxIdleConns int
+	ReadTimeout  string
+	WriteTimeout string
+}
+
+// DefaultRawDBConfig returns a default raw database config
+func DefaultRawDBConfig(readTimeout string) *RawDBConfig {
+	return &RawDBConfig{
+		ReadTimeout: readTimeout,
+	}
+}
+
+// AddWriteTimeout adds writeTimeout for raw database
+func (c *RawDBConfig) AddWriteTimeout(writeTimeout string) *RawDBConfig {
```

Comment:
`Set` or `Add`? same for `AddMaxIdleConns`.

Reply:
Done in [here](https://github.com/pingcap/dm/pull/266/files#diff-94b1e02e1fe1bd32d9eee7a40a730b00R62)